packages/devtools_app/lib/src/debugger/flutter/common.dart

```diff
@@ -45,3 +45,18 @@ Widget createAnimatedCircleWidget(double radius, Color color) {
     decoration: BoxDecoration(color: color, shape: BoxShape.circle),
   );
 }
+
+TextStyle regularStyle(BuildContext context) {
```

Comment:
consider making these extension methods on the context.

Reply:
Now that these have been moved out of the build methods, perhaps qualify the names slightly more - `regularTestStyle`?

Reply:
clean! good suggestion